<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>141</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 179, 255)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>741</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_taikhoan">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Quản lý tài khoản</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_mathang">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Quản lý mặt hàng</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_lichsu">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Lịch sử mua hàng</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>510</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Đăng xuất</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_magiamgia">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Mã giảm giá</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>671</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="taikhoan">
     <widget class="QTableWidget" name="bangTaiKhoan">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>461</width>
        <height>521</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20pt;</string>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <column>
       <property name="text">
        <string>email </string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>password</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>full_name</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="btn_addUser">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>0</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Thêm tài khoản</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="mathang">
     <widget class="QTableWidget" name="bangMatHang">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>571</width>
        <height>531</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 15pt</string>
      </property>
      <column>
       <property name="text">
        <string>Mã mặt hàng</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tên mặt hàng</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Số lượng</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Giá</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="lichsumua">
     <widget class="QTableWidget" name="tableWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>651</width>
        <height>531</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Mã hoá đơn</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Thời gian mua</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Mặt hàng mua</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tổng số tiền</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
